bundles/org.openhab.binding.pjlinkdevice/README.md

```diff
@@ -71,6 +76,45 @@ sitemap sample label="Main Menu" {
     Selection item=Projector_Input
     Switch item=Projector_AudioMute
     Switch item=Projector_VideoMute
+    Switch item=Projector_Lamp1Active
+    Text item=Projector_Lamp1Hours
   }
 }
 ```
+
+### Multiple lamps
+
+Most of the time, there's just one lamp. In case a projector has more than one lamp, additional channels for those lamps can be configured.
+
+sample-lamp-2.things:
+
+```
+pjLinkDevice:pjLinkDevice:MyProjector [ ipAddress="192.168.178.10" ]
+{
+  Channels:
+    Type lampHours : lamp2Hours "Lamp 2 Hours" [
+        lampNumber=2
```

Comment:
It looks like the value is not automatically validated against the min/max values declared in thing-types.xml, so I guess I'll have to do that from my code similar to the thing configuration [as discussed here](https://github.com/openhab/openhab2-addons/pull/3834#discussion_r315806865)?

Reply:
If this is not the case, it is a bug, too. IMO a binding should rely on value validation of the framework.

Reply:
@nils This is my only question left.

Reply:
No, but I confirmed that validation for channel parameters does not happen automatically if the thing is configured in a .things file. Should I create an issue for that in openhab-core?